examples/proc_ODNP_rough.py

```diff
@@ -35,7 +35,6 @@ def thetime(x, position):
     data.reorder(['power','t2'],first=False)
     fl.next('raw data')
     fl.image(data)
-    fl.show();quit()
```

Comment:
literally just taking out the quit statement, this script is unrelated to the PR but came up when JF searched "quit" in this repo.